### PR TITLE
LRDOCS-5852 update adding a dxp data source

### PR DESCRIPTION
**Pull request by jhinkey**
I updated the prerequisites and steps for adding a data source. Also updated the project welcome page pic.

### CODE REVIEW

File: discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown [hunk 1]
```diff
@@ -50,41 +50,53 @@ data source! It's time to add your Liferay DXP instance as a data source.
...
+    In Liferay DXP 7.1, you can find the Client ID and Secret by navigating to 
+    *Control Panel* &rarr; *Configuration* &rarr; *OAuth 2 Admin*.
 
-    - *URL:* The Liferay DXP instance URL.
-    - *Name:* A name for your data source. 
+    In Liferay DXP 7.0, you can find the Consumer Key and Secret on the *OAuth
+    Admin* page of the *Control Panel*. 
 
-    *Configuration Options*
+5.  Click *Authorize*. A window appears and asks if you want to authorize
```
Comment (christianstokes) [inline discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown]: The window will ask the user to login. They will need to login with a user who has the Admin role in the DXP server they are attempting to connect

File: discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown [hunk 2]
```diff
@@ -50,41 +50,53 @@ data source! It's time to add your Liferay DXP instance as a data source.
...
 
-    *Configuration Options*
+5.  Click *Authorize*. A window appears and asks if you want to authorize 
+    Analytics Cloud. 
 
-    - *Configure All:* Launches a wizard for configuring assets, touchpoints, 
-    and contacts from the instance. 
-    - *Configure Contacts:* Configures the contact data only.
-    - *Configure Analytics:* Configures the assets and touchpoints only.
+6.  Click *Authorize*. You've advanced to the *Configure Data Source* tab.
```
Comment (christianstokes) [inline discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown]: After they "Authorize" in the pop up, they will need to click "Save" to advance to the Configure screen

File: discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown [hunk 3]
```diff
@@ -50,41 +50,53 @@ data source! It's time to add your Liferay DXP instance as a data source.
...
-    and contacts from the instance. 
-    - *Configure Contacts:* Configures the contact data only.
-    - *Configure Analytics:* Configures the assets and touchpoints only.
+6.  Click *Authorize*. You've advanced to the *Configure Data Source* tab. 
 
-7.  Name your data source.
+Here are the configuration options:
 
-8.  Click *Configure All*. The Liferay DXP data source wizard appears, showing
+- *Configure All:* Launches a wizard for configuring assets, touchpoints,
```
Comment (christianstokes) [inline discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown]: This has been removed, the only options are Configure Contacts and Configure Analytics now

File: discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown [hunk 4]
```diff
@@ -50,41 +50,53 @@ data source! It's time to add your Liferay DXP instance as a data source.
...
-    - *Configure Analytics:* Configures the assets and touchpoints only.
+6.  Click *Authorize*. You've advanced to the *Configure Data Source* tab. 
 
-7.  Name your data source.
+Here are the configuration options:
 
-8.  Click *Configure All*. The Liferay DXP data source wizard appears, showing
+- *Configure All:* Launches a wizard for configuring assets, touchpoints, 
+and contacts from the instance. 
+- *Configure Contacts:* Configures the contact data only.
```
Comment (christianstokes) [inline discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown:98]: I will send you screenshots for everything below here. It is mostly the same except there might be a few slight variations in the screenshots and UI

Comment (jhinkey) [reply discover/analytics-cloud/articles/02-getting-started/02-adding-a-liferay-dxp-data-source.markdown:98]: Awesome! They've been very helpful!!

**Comment by jhinkey** [comment]
I incorporated your suggestions and updated the rest of the steps. Please review. Thanks!

**Pull request closed without merging**